packages/blade/src/components/Button/BaseButton/StyledBaseButton.d.ts

```diff
@@ -0,0 +1,25 @@
+import type { Theme } from '../../BladeProvider';
+import type { BaseButtonProps } from './BaseButton';
+import type { ButtonMinHeight } from './buttonTokens';
+
+export type StyledBaseButtonProps = Omit<BaseButtonProps, 'icon' | 'children'> & {
+  activeBorderColor: string;
+  activeBackgroundColor: string;
+  defaultBorderColor: string;
+  minHeight: `${ButtonMinHeight}px`;
+  spacing: `${ValueOf<Theme['spacing']>}px ${ValueOf<Theme['spacing']>}px`;
```

Comment:
this might work here but later we might need to have a makeSpace equivalent at Type level basis that the types would return px/rem